module-apps/application-messages/data/MessagesStyle.hpp

```diff
@@ -106,6 +106,14 @@ namespace style
             inline constexpr uint32_t w = style::listview::body_width_with_scroll;
         } // namespace smsList
 
+        namespace oldSmsLabel
+        {
+            inline constexpr uint32_t x = style::messages::smsList::x;
```

Comment:
You can use `std::uint32_t` :)

Reply:
Even better: if types match, just use `constexpr auto`. If they don't, still use `constexpr auto` and an explicit cast to avoid implicit conversions :)